Chat/TokenManagement/UserTokenManager.cs

```diff
@@ -28,5 +29,26 @@ public async Task<(string userMri, string skypeToken, long expiresDateUtc)> Gene
                 return (null, null, -1);
             }
         }
+
+        public async Task<(string userMri, string skypeToken, long expiresDateUtc)> RefreshTokenAsync(string resourceConnectionString, string identity)
```

Comment:
`MRI` and `Skype` are two words that shouldn't show up in our public Azure Communication code.

```suggestion
        public async Task<(string userIdentity, string token, long expiresDateUtc)> RefreshTokenAsync(string resourceConnectionString, string identity)
```